UKHomeOffice/sgmr-service
# SPCR-10 Add info page to submit voyage flow

## Description
- **Given** I am on the /voyage-plans page
- **When** I click on start now
- **Then** I am taken to a page with the content attached
<img width="734" alt="Screenshot 2022-01-07 at 14 50 36" src="https://user-images.githubusercontent.com/56563329/149544016-e3c98a0b-2a03-4180-bb75-585ef9842fe6.png">

## To Test
- Go to the Reports/Voyage Plan Dashboard
- Click on Start now
- You should be taken to the page with the content attached
- When you click Cancel it should take you back to the Reports/Voyage Plan Dashboard
- When you click Save and Continue it should take you to the Departure details section of the form

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
